crates/core/tedge_actors/src/runtime.rs

```diff
@@ -305,7 +304,8 @@ mod tests {
             while let Some(message) = self.messages.recv().await {
                 match message {
                     EchoMessage::String(message) => {
-                        self.messages.send(EchoMessage::String(message)).await?
+                        crate::Sender::send(&mut self.messages, EchoMessage::String(message))
```

Comment:
Had to do this due to conflicting `Sender` from `futures_channel` crate in the same scope.